source/fundamentals/crud/write-operations/bulk.txt

```diff
@@ -324,43 +329,96 @@ Client Bulk Write
 When connecting to a deployment running {+mdb-server+} 8.0 or later,
 you can use the ``MongoClient.bulkWrite()`` method to write
 to multiple databases and collections in the same cluster. The ``MongoClient.bulkWrite()``
-method does not split different kinds of operations into different batches.
+method performs all write operations in a single call.
 
-In the examples in preceding sections of this page, the ``MongoCollection.bulkWrite()`` method 
-takes a list of ``WriteModel`` instances in which the specified subclass of 
-``WriteModel`` represents the corresponding write operation. For example, an
-instance of ``InsertOneModel`` represents an operation to insert one document. 
-
-Similarly, the ``MongoClient.bulkWrite()`` method takes a 
+The ``MongoClient.bulkWrite()`` method takes a 
 list of ``ClientNamespacedWriteModel`` instances to represent different write operations.
-For example, an instance of ``ClientNamespacedInsertOneModel`` represents an
-operation to insert one document.
+You can construct instances of the ``ClientNamespacedWriteModel`` interface by using 
+instance methods. For example, an instance of ``ClientNamespacedInsertOneModel`` represents an
+operation to insert one document, and you can create this model by using
+the ``ClientNamespacedWriteModel.insertOne()`` method.
+
+The models and their corresponding instance methods are described
+in the table below.
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Model
+     - Instance Method
+     - Description
+     - Parameters
+
+   *  - ``ClientNamespacedInsertOneModel``
+      - ``insertOne()``
+      - Creates a model to insert a document into the ``namespace``.
+      - ``namespace``: defines which database and collection to write to
+       
+        ``document``: defines the document to insert
```

Comment:
For all of these parameters, removes 'defines which' or 'defines a' and then capitalize the first letter (e.g., 'Database and collection to write to')

https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items